internal/pkg/override/errors.go

```diff
@@ -7,17 +7,30 @@ import (
 	"fmt"
 )
 
-type errNPMUnavailable struct {
-	parent error
+type errorExecutableNotFound struct {
+	executable string
+	error      error
+}
+
+func (e *errorExecutableNotFound) Error() string {
+	return fmt.Sprintf("look up %q: %s", e.executable, e.error.Error())
+}
+
+type errPackageManagerUnavailable struct {
+	parentError error
 }
 
-func (err *errNPMUnavailable) Error() string {
-	return fmt.Sprintf(`"npm" cannot be found: "npm" is required to override with the Cloud Development Kit: %v`, err.parent)
+func (err *errPackageManagerUnavailable) Error() string {
+	return fmt.Sprintf("cannot find a package manager to override with the Cloud Development Kit:\n%s", err.parentError.Error())
```

Comment:
nit - i think this is the case? does it only work with javascript package managers?
```suggestion
	return fmt.Sprintf("cannot find a javascript package manager to override with the Cloud Development Kit:\n%s", err.parentError.Error())
```